### PR TITLE
Twitter request api #4

### PR DESCRIPTION
**Pull request by Eik-S**
## Changes:
  * **/conftest.py:** Config file for pytest, atm just defining its root directory with being present
  * **/requirements.txt:** Added pytest dependency
  * **/docker-compose.yml:** Added ignored .env file for twitter credentials necessary for developing
  * **/api-tests/:** Test folder with same structure as **/api/**
  * **Travis Env:** Added same twitter credentials as env variables to travis ci settings, used for testing

## Resources:
  * [pytest docs](https://docs.pytest.org/en/latest/getting-started.html#getstarted)

### CODE REVIEW

File: api/twitter/twitterApi.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    user_dict[ "lang"] = str(data['lang'])
+    user_dict[ "followers_count"] = int(data['followers_count'])
+    user_dict[ "friends_count"] = int(data['friends_count'])
+    user_dict[ "statuses_count"] = int(data['statuses_count'])
+    user_dict[ "created_at"] = str(data['created_at'])
+    user_dict[ "profile_img_url"] = str(data['profile_image_url'])
+    return( response)
+
+
+def get_user_info( uid):
```
Comment (Bacis) [inline api/twitter/twitterApi.py]: If this is an API this should be a flask endpoint.

File: api/twitter/twitterApi.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    """Request Useer Information, return a dictionary"""
```
Comment (Bacis) [inline api/twitter/twitterApi.py]: Spelling

File: api/twitter/twitterApi.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    else:
+        print("Invalid data, need string or list at get_user_info")
+        return ""
+
+def get_retweeters( tweetId):
+    """Request the 100 last retweet ids, return them as a list"""
+    data = api.request('statuses/retweeters/ids', { 'id': str(tweetId)})
+    return data.json()['ids']
+
+print(get_retweeters("935929980697145346"))
```
Comment (Bacis) [inline api/twitter/twitterApi.py]: I wouldn't hard code this here, make it modular, pass the GET parameter with flask endpoints

File: api/twitter/twitterApi.py (hunk 2 above)
Comment (Eik-S) [reply api/twitter/twitterApi.py]: corrected

File: api/twitter/twitterApi.py (hunk 3 above)
Comment (Eik-S) [reply api/twitter/twitterApi.py]: deleted, just forgot testing request.

**Comment by Eik-S** [comment]
Will rewrite the functions to flask endpoints if necessary with the next merge.

**Pull request merged**